docs/unified/novacustom/initial-deployment.md

```diff
@@ -0,0 +1,145 @@
+# Initial Deployment
+
+## Intro
+
+This document is a guide for the initial installation of Dasharo on a supported
+device. It assumes some knowledge about external flashing and is primarily aimed
+at technicians performing initial installation and not for end users of the
+devices.
+
+## Installing Dasharo
+
+### Initial Installation
+
+=== "12th Gen (Alder Lake)"
+
+    During the initial installation of Dasharo, you should deploy the supported
+    Intel ME version (and configuration) on the device. Since vendor firmware
+    has enabled Intel Boot Guard and BIOS Guard, it is not possible to do this
+    from within the operating system and external flashing of the whole flash
+    chip using a programmer like the CH341a or
+    [3mdeb RTE](http://docs.dasharo.com/transparent-validation/rte/introduction/)
+    is required.
+
+    > Publicly released binaries do not contain ME binary. If you need an Intel ME
+    > update for your device, contact us via already established commercial support
+    > channel.
+
+    ### Preparation
+
+    Install flashrom:
+
+    ```bash
+    sudo apt -y install flashrom
+    ```
+
+    ### Installation
+
+    Steps for initial Dasharo installation:
+
+    1. Open the laptop.
+
+        === "NS5x / NS7x"
+            ![ns5x chips](../../images/ns50mu_board_chips.jpg)
+
+        === "NV4x"
+            ![nv4x chips](../../images/nv4x_board_chips.jpg)
+
+    1. Disconnect the primary battery. (1)
+    1. Disconnect the CMOS battery. (2)
+    1. Attach a WSON-8 probe to the SPI flash chip. (3)
+    1. Execute the following command, replacing [path] with the path to the firmware
+        image you want to flash, e.g. `novacustom_ns5xpu_ns7xpu_full_v1.0.0.rom`
+
+        ```bash
+        flashrom -p ch341a_spi -w [path]
+        ```
+
+    1. Detach the WSON-8 probe.
+    1. Connect the primary battery and reconnect the CMOS battery
+    1. Power on the laptop. The laptop may shut down once after training the memory.
+
+    ### Transition to open EC
+
+    If you have deployed the Dasharo using external programmer, time to switch to
+    Open EC. This step is required to support all Dasharo features. To install Open
+    EC Firmware on NS5xPU/NS7xPU, follow this instruction:
+    [Dasharo EC Transition](../../../common-coreboot-docs/dasharo_tools_suite/#dasharo-ec-transition).
```

Comment:
Updated. Added a manual installation guide using Dasharo flashrom fork.

Reply:
@mkopec Wouldn't it be simpler to `wget` this script https://github.com/Dasharo/dasharo-tools/pull/2 than to disconnect the battery?

Reply:
Also, at this point we can already use DTS over network as we have Dasharo flashed. 
Your document probably assumes Ubuntu, although it is not mentioned anywhere in the document what OS is used.

Reply:
That way we remove most of the section (building and installing flashrom)
1. Instruct to run DTS - link to relevant docs
2. Instruct to drop to shell, wget and run script.

Reply:
> @mkopec Wouldn't it be simpler to wget this script https://github.com/Dasharo/dasharo-tools/pull/2 than to disconnect the battery?

Sure, but the script depends on `iotools` which isn't available in the Ubuntu repo - which is why I left a TODO to use /dev/port directly instead of `iotools` to talk to the EC

Reply:
But do we need to use Ubuntu? Why not use DTS if the Dasharo is already flashed and we can simply enter boot menu entry for DTS?

Reply:
yes @TomaszAIR will do that

Reply:
I created issue for that https://github.com/Dasharo/dasharo-issues/issues/255, it will be added in next DTS release.